### PR TITLE
Add note about specifying minor for corejs property

### PR DESCRIPTION
**Pull request by ewanharris**
When specifying just `3` it can lead to a situation where expected polyfills are missing due to changes released in a core-js minor version, e.g. `Promise.allSettled` was moved to stable, but won't be polyfilled by `corejs: 3` but will by `corejs: 3.2`.

By specifying the corejs version in a `3.minor` fashion it ensures that the right polyfills are imported. This is already stated in the core-js docs, but it should be mirrored in the docs for preset-env

References:

core-js issue: zloirock/core-js/issues/793
core-js docs: https://github.com/zloirock/core-js#babelpreset-env

### CODE REVIEW

File: docs/preset-env.md [hunk 1]
```diff
@@ -392,7 +392,9 @@ Don't add polyfills automatically per file, and don't transform `import "core-js
 
 Added in: `v7.4.0`
 
-`2`, `3` or `{ version: 2 | 3, proposals: boolean }`, defaults to `2`.
+`2`, `3`, `3.minor`, or `{ version: 2 | 3, proposals: boolean }`, defaults to `2`.
```
Comment (ewanharris) [inline docs/preset-env.md:395]: This probably isn't the best way to document it, but I couldn't figure out a better way

Comment (nicolo-ribaudo) [reply docs/preset-env.md:395]: Maybe we can just put "`string` or `{ version: string, proposals: boolean }` here, and then list the possible versions more in prose?

Comment (JLHwung) [reply docs/preset-env.md:395]: We should recommend string, e.g. `"3.10"`, for minor version usage here, otherwise we can't differentiate 3.1 and 3.10.

Comment (ewanharris) [reply docs/preset-env.md:395]: I'll update it to state `string` or `{ version: string, proposals: boolean }` and then update the note to expand upon that

**Review by JLHwung — APPROVED** [review]

**Comment by ewanharris** [comment]
This was ultimately done by #2438 so closing

**Pull request closed without merging**